setup.sh

```diff
@@ -0,0 +1,24 @@
+#! /usr/bin/env bash
+
+OLD_PWD=$(pwd)
+
+# Additional ZShell configs
+ADDITIONAL_CONFIGS='source ~/.config/zshrc/*'
+if [[ ! $( grep "$ADDITIONAL_CONFIGS" ~/.zshrc ) ]] ; then
+  echo "Adding custom additional configs from $ADDITIONAL_CONFIGS to ~/.zshrc" ;
+  echo $ADDITIONAL_CONFIGS >> ~/.zshrc ;
```

Comment:
There are few things worse than scripts that mess with my `.zshrc`. But then again, YMMV.